buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -452,6 +454,41 @@ class VagrantTestPlugin implements Plugin<Project> {
                 command BATS_TEST_COMMAND
             }
             packagingTest.dependsOn(packaging)
+
+            String task = "vagrant${boxTask}#repositoryTest"
+
+            String expectedRepoVersion = project.getProperties().get('repo.version')
+            String url = project.getProperties().get('repo.url')
+
+            Task createUrlFileTask = project.tasks.create("vagrant${boxTask}#createUrlFileTask", FileContentsTask) {
+                file "${batsDir}/archives/repo_url"
```

Comment:
I basically copied this from [here](https://github.com/elastic/elasticsearch/blob/master/buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy#L247) and [here](https://github.com/elastic/elasticsearch/blob/master/buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy#L253)

should we fix that as well?